quesma/model/typical_queries/hits.go

```diff
@@ -132,12 +133,19 @@ func (query Hits) addAndHighlightHit(hit *model.SearchHit, resultRow *model.Quer
 	}
 }
 
+func (query Hits) WithoutTimestampField() Hits {
+	query.useDefaultID = true
+	return query
+}
+
 func (query Hits) computeIdForDocument(doc model.SearchHit, defaultID string) string {
-	tsFieldName, err := query.table.GetTimestampFieldName()
-	if err != nil {
+
+	if query.useDefaultID {
 		return defaultID
 	}
 
+	tsFieldName := model.TimestampFieldName
```

Comment:
As we've chatted on Zoom, this isn't correct as it always takes `@timestamp`

Reply:
It is fixed now.